src/Storages/StorageMongoDB.h

```diff
@@ -51,7 +51,7 @@ class StorageMongoDB final : public ext::shared_ptr_helper<StorageMongoDB>, publ
     std::string username;
     std::string password;
 
-    Context global_context;
```

Comment:
Seems like it's never used.
@ageraab, why do we need it?

Reply:
I would not even ask... if it's unused, we don't need it.